proofzero/rollupid
# feat(profile): disconnect address in connected address screen

# Description

- [x] Disconnect action to connected accounts list
- [x] Disconnect modal
- [x] Disconnect endpoint under connections route
- [x] Disconnect method in galaxy account resolver
- [x] Validate account owns address to be disconnected
- [x] Validate address to be disconnected is not last one under account
- [x] Disconnect method in account gql
- [x] Hook unsetAccount from address client
- [x] Extract action modals into individual components

- Closes #1703 

## Type of change

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

1. Login from new account
* Notice no action to disconnect available
2. Connect new account
* Notice actions to disconnect available
3. Disconnect account
* Notice removal from account listing
* Notice no action to disconnect left account available
